en/docs/administer/key-managers/configure-forgerock-connector.md

```diff
@@ -0,0 +1,288 @@
+            <td>The endpoint that issues the access tokens.
+            </br> Example: </br>
+            http://localhost:8080/openam/oauth2/access_token</br>
+            </td>
+            <td>Mandatory</td>
+          </tr>
+          <tr class="odd">
+            <td>Revoke Endpoint</td>
+            <td>The endpoint that revokes the access tokens.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td>Userinfo Endpoint</td>
+            <td>The endpoint that allows clients to verify the identity of the end-user based on the authentication performed by an authorization server, as well as to obtain basic profile information about the end-user.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td>Authorize Endpoint</td>
+            <td>The endpoint that is used to obtain an authorization grant from the resource owner via user-agent redirection.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td>Scope Management Endpoint </td>
+            <td>The endpoint that is used to manage the scopes.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td><b>Connector Configurations</b></td>
+            <td></td>
+            <td></td>
+          </tr>
+          <tr class="even">
+            <td>Client ID</td>
+            <td>The client Id of the static client to invoke the introspection endpoint. </td>
+            <td>Mandatory</td>
+          </tr>
+          <tr class="odd">
+            <td>Client Secret</td>
+            <td>The client secret of the static client to invoke the introspection endpoint.</td>
+            <td>Mandatory</td>
+          </tr>
+          <tr class="even">
+            <td><b>Claim URIs</b></td>
+            <td>Provide the claim URIs for the consumer key and the scopes.</td>
+            <td></td>
+          </tr>
+          <tr class="odd">
+            <td>Consumer Key Claim URI</td>
+            <td>The claim URI for the consumer key.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td>Scopes Claim URI</td>
+            <td>The claim URI for scopes.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td>Grant Types</td>
+            <td>The supported grant types.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td><b>Certificates</b></td>
+            <td></td>
+            <td></td>
+          </tr>
+          <tr class="odd">
+            <td>PEM</td>
+            <td>Either copy and paste the certificate in PEM format or upload the PEM file.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td>JWKS</td>
+            <td>The JSON Web Key Set (JWKS) endpoint is a read-only endpoint. This URL returns PingFederate's public key set in JSON web key set format.
```

Comment:
Why does this state "PingFederate" when we are writing about ForgeRock? Looks like it is an error due to copying and pasting content.
This issue appears in multiple places.

Reply:
@shilmyhasan , this comment is still pending

Reply:
Will fix the relevant places